src/Simplex/Chat.hs

```diff
@@ -1549,10 +1621,12 @@ processChatCommand' vr = \case
     currentTs <- liftIO getCurrentTime
     withFastStore' $ \db -> setConditionsNotified db conditionsId currentTs
     ok_
-  APIAcceptConditions conditionsId operators -> do
+  -- TODO switch to IDs
```

Comment:
needs operator tags on insert, so if switched to ids it'd have to read operators/operator tags first, why not just pass operator objects